blocks/test/fixtures/core-embed.serialized.html

```diff
@@ -0,0 +1,6 @@
+<!-- wp:core/embed url="https://example.com/" -->
+<figure>https://example.com/
```

Comment:
Not particularly relevant to this pull request, but I imagine this won't display the embed correctly on the front-end? Probably gets mixed up that the URL isn't on its own line.